Sources/Algorithms/Unique.swift

```diff
@@ -30,8 +39,23 @@ extension Sequence where Element: Hashable {
 
 extension Sequence {
   /// Returns an array with the unique elements of this sequence (as determined
-  /// by the given projection), in the order of the first occurence of each
+  /// by the given projection), in the order of the first occurrence of each
   /// unique element.
+  ///
+  ///     let animals = ["dog", "pig", "cat", "ox", "cow", "owl"]
+  ///     let uniqued = animals.uniqued(on: {$0.first})
```

Comment:
Let's give an introduction to this example, as well:

```suggestion
  /// This example finds the elements of the `animals` array with unique
  /// first characters:
  ///
  ///     let animals = ["dog", "pig", "cat", "ox", "cow", "owl"]
  ///     let uniqued = animals.uniqued(on: { $0.first })
```